build/install_tools.sh

```diff
@@ -15,9 +15,9 @@ $(go env GOPATH)/bin/golangci-lint version | grep ${GOLNT_VERSION} || curl -sSfL
 
 if [[ "${OSTYPE}" == "linux"* ]]; then
   /tmp/typos --version | grep ${TYPOS_VERSION} || wget -qO- https://github.com/crate-ci/typos/releases/download/v${TYPOS_VERSION}/typos-v${TYPOS_VERSION}-x86_64-unknown-linux-musl.tar.gz | tar -zxf - -C /tmp/ ./typos
+  sudo snap install mdl
```

Comment:
Moved this command here because on MacOS mdl won't work even if you manually install snap. This is what I get after `sudo snap install mdl` is executed:
`Interacting with snapd is not yet supported on darwin.`

Reply:
thanks